docs/reference/backend/user/create-user.mdx

```diff
@@ -238,6 +238,19 @@ const response = await clerkClient.users.createUser({
 })
 ```
 
+## Example
+
+<Include src="_partials/create-user" />
+
 ## Backend API (BAPI) endpoint
 
 This method in the SDK is a wrapper around the BAPI endpoint `POST/users`. See the [BAPI reference](/docs/reference/backend-api/tag/users/post/users){{ target: '_blank' }} for more information.
+
+Here's an example of making a request directly to the endpoint using cURL.
+
+```bash {{ filename: 'terminal' }}
+  curl 'https://api.clerk.com/v1/users' -X POST -H 'Authorization:Bearer {{secret}}' -H 'Content-Type:application/json' -d '{
+    "email_address": ["test@example.com"],
+    "password": "my-secure-password"
+  }'
+```
```

Comment:
Should this have the `<SignedIn>` + "already injected" and `<SignedOut>` + "replace `YOUR_SECRET_KEY`" callouts?